src/presence_analyzer/templates/base.html

```diff
@@ -8,44 +8,59 @@
     <meta name="author" content="STX Next sp. z o.o."/>
     <meta name="viewport" content="width=device-width; initial-scale=1.0">
 
-    <link href="{{ url_for('static', filename='css/normalize.css') }}" media="all" rel="stylesheet" type="text/css" />
-    <link href="{{ url_for('static', filename='css/base.css') }}" rel="stylesheet" type="text/css" />
+    <link href="${url_for('static', filename='css/normalize.css')}" media="all" rel="stylesheet" type="text/css" />
+    <link href="${url_for('static', filename='css/base.css')}" rel="stylesheet" type="text/css" />
 
-    <script src="{{ url_for('static', filename='js/jquery.min.js') }}"></script>
+    <script src="${url_for('static', filename='js/jquery.min.js')}"></script>
     <script type="text/javascript" src="https://www.google.com/jsapi"></script>
-    {% block script %}{% endblock %}
+    <%block name='script'></%block>
 
 </head>
 <body>
     <div id="main">
         <div id="header">
             <h1>Presence analyzer</h1>
             <ul>
-                <li {% if request.path == '/presence_weekday' %}id="selected"{% endif %}>
-                    <a href="{{ url_for('template_router', path='presence_weekday') }}">Presence by weekday</a>
+                % if request.path == '/presence_weekday':
```

Comment:
This can be a for loop instead of 3 separate `if`s. You can create a list of paths compared in loop with `request.path`.